internal/ip/ip_darwin.go

```diff
@@ -61,3 +67,8 @@ func getIPHeaderLayerV4(tos uint8, tcpLen uint16, srcIP net.IP, dstIP net.IP) *l
 
 	return header
 }
+
+// attachBPF will attach an assembled BPF filter to the recvSource's raw socket file descriptor
+func (r *recvSource) attachBPF(filter []bpf.RawInstruction) error {
+	return errors.New("BPF Filter not supported on Darwin")
```

Comment:
Add `currently`? Otherwise, it is implied that BPF is not supported on Darwin in general...